README.rst

```diff
@@ -7,6 +7,24 @@ Documentation on this site
 *******************************
 
 
+Installing PyIMCOM
+-----------------------------
```

Comment:
These dashes, and the ones above and below as well, should be same as the length of the line above them.